src/main/java/org/folio/rest/impl/UserTenantsAPI.java

```diff
@@ -92,5 +96,5 @@ private void addWhereClauseArgumentsToCriterion(ArgumentsHolder argumentsHolder,
       .forEach(param -> criterion.addCriterion(param, queryOp));
   }
 
-  record ArgumentsHolder(String userId, String username, String tenantId, String email, String phoneNumber, String mobilePhoneNumber) {}
+  record ArgumentsHolder(String userId, String username, String tenantId, String email, String phoneNumber, String mobilePhoneNumber, String barcode, String externalSystemId) {}
```

Comment:
Do we have any code style rules in this module? In this case long line creates horizontal scroll.
```
  record ArgumentsHolder(
    String userId,
    String username,
    String tenantId,
    String email,
    String phoneNumber,
    String mobilePhoneNumber,
    String barcode,
    String externalSystemId
  ) {
  }
```

Reply:
Updated